src/test/java/web/tests/FlipkartTest.java

```diff
@@ -166,4 +166,22 @@ public void testAddToCartFunctionality(String product) {
         }
         softAssert.assertThat(Integer.parseInt(cartPage.getShoePriceTotalInCart()) == totalPriceOfShoeSelectedFromList).isTrue();
     }
+
+    /**
+     *
+     * @param item :accepting shoes from json
+     * @param position :accepting position i.e to select 2nd position shoes
+     */
+    @ParameterizedTest
+    @CsvFileSource(resources = "/testProductPageDesign.csv")
+    public void testProductPageDesign(String item,int position){
+        ProductPage productPage = new HomePage().popUpCancel().setShoes(item).searchShoes()
+                .OpenProductPage(position);
+        productPage.scrollToBottom();
+        ArrayList<String> headersInSelectionPage  = new ArrayList<>(Arrays.asList(map.get("headers").split(",")));
+        for (int row = 0; row < headersInSelectionPage.size(); row++) {
+            softAssert.assertThat(productPage.isSelectionDisplayed(headersInSelectionPage.get(row))).
+                    as("Json selected Headers matched with selection page Headers").isTrue();
```

Comment:
Shouldn't the description be the opposite? - 'Json selected Headers is NOT matching with selection page Headers'
Please note that this description only gets displayed when the assertion fails.